leader/src/main/kotlin/no/nav/familie/leader/LeaderClient.kt

```diff
@@ -8,25 +9,54 @@ import java.net.http.HttpResponse
 import java.net.http.HttpResponse.BodyHandlers
 
 object LeaderClient {
+    private val logger = LoggerFactory.getLogger(LeaderClient::class.java)
+
     /**
      * @return Om pod er leader eller ikke. null hvis leadersjekk ikke er implementert på pod
      */
     @JvmStatic
-    fun isLeader(): Boolean? {
-        val electorPath = Environment.hentLeaderSystemEnv() ?: return null
+    fun isLeader(
+        antallGanger: Int = 3,
+        forsinkelseIms: Long = 1000,
```

Comment:
Kan vurdere å sende inn en variabel av type `Duration` da `Thread.sleep()`også godtar det

```suggestion
        forsinkelse Duration = Duration.ofMillis(1000),
```